yuuki/proxy.py

```diff
@@ -43,31 +44,68 @@ def recieve():
     """
     if not request.json:
         abort(400)
-    
+
     response = PROFILE.dispatch(request.get_json())
     return jsonify(response), 200
 
+def parse_config(config_file):
+
+    """
+    Receive a file path and parse it to a dict
+    """
+
+    config_reader = SafeConfigParser()
+    config_reader.read(config_file)
+    config_dict = {}
+
+    for section_name in config_reader.sections():
+
+        config_dict[section_name] = {}
+
+        for name, value in config_reader.items(section_name):
+
+            config_dict[section_name][name] = value
+    
+    return config_dict
 
 def main():
+
     """
     Parse configuration and start flask app.
-
-    WARNING: only a single profile file is supported at this time
     """
+    
     parser = argparse.ArgumentParser()
-    parser.add_argument('--port', type=int, default=9001,
-                        help="port to listen on (default=9001)")
-    parser.add_argument('profiles', nargs='+',
-                        help="full path to OpenC2 profile")
+
+    parser.add_argument('--conf', type=str, default="yuuki.conf",help="Location of yuuki.conf")
 
```

Comment:
Configuration files are great, but I'd like to keep the option for command line switches. I've found it very useful for debugging. I think this should do it:

group = parser.add_mutually_exclusive_group(required=True)
group.add_argument('--conf', default=None)
group.add_argument('--profile', nargs='+', default=None)

args = parser.parse_args()

if args.profile:
    if not args.port:
        args.port = 9001

    # configuration from command line args

else:
    # configuration from configuration file